README.md

```diff
@@ -75,7 +91,10 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza = {temperature : 70,
```

Comment:
Almost! This would throw an error. [Take a look at the solution.](https://github.com/ga-wdi-quizzes/week-02/tree/solution#question-6)